spec/integration/outputs/retry_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:max_retries) { 3 }
 
   def mock_actions_with_response(*resp)
-    LogStash::Outputs::Elasticsearch::HTTPClient
+    LogStash::Outputs::Elasticsearch::HttpClient
```

Comment:
@jordansissel 

![I have no strong feelings about this](http://www.quickmeme.com/img/df/df1a536c46e7aeccc536212426c90ae2a37f3555e78be5419850b3d0f0149d45.jpg)